pkg/downloads/versions.go

```diff
@@ -44,27 +44,19 @@ var GithubCommitSha1 string
 
 // GithubRepository represents the value of the "GITHUB_CHECK_REPO" environment variable
 // Default is "elastic-agent"
-var GithubRepository string
+var GithubRepository string = "elastic-agent"
```

Comment:
@v1v we are setting this value as default because the tests are mainly focused on the agent scenarios. Those for beats are just a few including filebeat/metricbeat